File: client/iih_stub/iih.ci_stub/iih.ci.mrqry/mrtplusagerate/d/MrTplUsageRateDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mr.d
{
    /// <summary>
    /// MrTplUsageRate 的摘要说明。
    /// </summary>
    public class MrTplUsageRateDTO : BaseDTO {

        public MrTplUsageRateDTO() {
 
        }

        /// <summary>
        /// 模板名称
        /// </summary>
		public string Name_template {
            get { return getAttrVal<string>("Name_template",null); }
            set { setAttrVal<string>("Name_template", value); }
        }

        /// <summary>
        /// 使用次数
        /// </summary>
		public string Count_usage {
            get { return getAttrVal<string>("Count_usage",null); }
            set { setAttrVal<string>("Count_usage", value); }
        }

        /// <summary>
        /// 门诊/住院记录总数
        /// </summary>
		public string Count_total {
            get { return getAttrVal<string>("Count_total",null); }
            set { setAttrVal<string>("Count_total", value); }
        }

        /// <summary>
        /// 使用率
        /// </summary>
		public string Rate_usage {
            get { return getAttrVal<string>("Rate_usage",null); }
            set { setAttrVal<string>("Rate_usage", value); }
        }

        /// <summary>
        /// 就诊类型
        /// </summary>
		public string Name_entp {
            get { return getAttrVal<string>("Name_entp",null); }
            set { setAttrVal<string>("Name_entp", value); }
        }

        /// <summary>
        /// 模板分类
        /// </summary>
		public string Name_doctype {
            get { return getAttrVal<string>("Name_doctype",null); }
            set { setAttrVal<string>("Name_doctype", value); }
        }

        /// <summary>
        /// 科室名称
        /// </summary>
		public string Name_dep {
            get { return getAttrVal<string>("Name_dep",null); }
            set { setAttrVal<string>("Name_dep", value); }
        }

        /// <summary>
        /// 创建人
        /// </summary>
		public string Name_mrtp_create {
            get { return getAttrVal<string>("Name_mrtp_create",null); }
            set { setAttrVal<string>("Name_mrtp_create", value); }
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id_mr_tpl_ur {
            get { return getAttrVal<string>("Id_mr_tpl_ur",null); }
            set { setAttrVal<string>("Id_mr_tpl_ur", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Name_template", "Count_usage", "Count_total", "Rate_usage", "Name_entp", "Name_doctype", "Name_dep", "Name_mrtp_create", "Id_mr_tpl_ur"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.d.MrTplUsageRate";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/d/MrTplUsageRateDTO.java
package iih.ci.mr.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 病历模板使用率 DTO数据 
 * 
 */
public class MrTplUsageRateDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 模板名称
	 * @return String
	 */
	public String getName_template() {
		return ((String) getAttrVal("Name_template"));
	}
	/**
	 * 模板名称
	 * @param Name_template
	 */
	public void setName_template(String Name_template) {
		setAttrVal("Name_template", Name_template);
	}
	/**
	 * 使用次数
	 * @return String
	 */
	public String getCount_usage() {
		return ((String) getAttrVal("Count_usage"));
	}
	/**
	 * 使用次数
	 * @param Count_usage
	 */
	public void setCount_usage(String Count_usage) {
		setAttrVal("Count_usage", Count_usage);
	}
	/**
	 * 门诊/住院记录总数
	 * @return String
	 */
	public String getCount_total() {
		return ((String) getAttrVal("Count_total"));
	}
	/**
	 * 门诊/住院记录总数
	 * @param Count_total
	 */
	public void setCount_total(String Count_total) {
		setAttrVal("Count_total", Count_total);
	}
	/**
	 * 使用率
	 * @return String
	 */
	public String getRate_usage() {
		return ((String) getAttrVal("Rate_usage"));
	}
	/**
	 * 使用率
	 * @param Rate_usage
	 */
	public void setRate_usage(String Rate_usage) {
		setAttrVal("Rate_usage", Rate_usage);
	}
	/**
	 * 就诊类型
	 * @return String
	 */
	public String getName_entp() {
		return ((String) getAttrVal("Name_entp"));
	}
	/**
	 * 就诊类型
	 * @param Name_entp
	 */
	public void setName_entp(String Name_entp) {
		setAttrVal("Name_entp", Name_entp);
	}
	/**
	 * 模板分类
	 * @return String
	 */
	public String getName_doctype() {
		return ((String) getAttrVal("Name_doctype"));
	}
	/**
	 * 模板分类
	 * @param Name_doctype
	 */
	public void setName_doctype(String Name_doctype) {
		setAttrVal("Name_doctype", Name_doctype);
	}
	/**
	 * 科室名称
	 * @return String
	 */
	public String getName_dep() {
		return ((String) getAttrVal("Name_dep"));
	}
	/**
	 * 科室名称
	 * @param Name_dep
	 */
	public void setName_dep(String Name_dep) {
		setAttrVal("Name_dep", Name_dep);
	}
	/**
	 * 创建人
	 * @return String
	 */
	public String getName_mrtp_create() {
		return ((String) getAttrVal("Name_mrtp_create"));
	}
	/**
	 * 创建人
	 * @param Name_mrtp_create
	 */
	public void setName_mrtp_create(String Name_mrtp_create) {
		setAttrVal("Name_mrtp_create", Name_mrtp_create);
	}
	/**
	 * 主键
	 * @return String
	 */
	public String getId_mr_tpl_ur() {
		return ((String) getAttrVal("Id_mr_tpl_ur"));
	}
	/**
	 * 主键
	 * @param Id_mr_tpl_ur
	 */
	public void setId_mr_tpl_ur(String Id_mr_tpl_ur) {
		setAttrVal("Id_mr_tpl_ur", Id_mr_tpl_ur);
	}
}